openshift-pipelines/images/art-jenkins-slave.Dockerfile

```diff
@@ -22,5 +22,6 @@ RUN yum install -y \
         rhpkg \
     && yum clean all
 
-RUN pip3 install -U koji tox twine requests>=2.20 setuptools wheel codecov rh-doozer rh-elliott rh-ocp-build-data-validator
+# TODO: Clarify: Is this the image used to run rhcos_sync in prod?
```

Comment:
Will lift the hold on this PR once this question is addressed.

Reply:
yes

Reply:
I've ping'd ART folks asking if they can confirm.

Reply:
@LorbusChris based on the response from ART are we OK with moving this forward?

Reply:
@ashcrow this is blocked on https://issues.redhat.com/browse/ART-2597

Reply:
cc'ing @thegreyd who's assigned to it :) I guess installing `awscli` in this Dockerfile isn't strictly required for now - should I remove it or leave it here?

Reply:
Hey @LorbusChris :wave: I think you can leave this out and instead add the dependency in the [build.groovy file for the job](https://github.com/openshift/aos-cd-jobs/blob/6c05096c4be1bdf327fd8aef628ec4cf2a55c781/jobs/build/rhcos_sync/build.groovy#L58), something like `commonlib.shell(script: "pip install awscli")`, each of our job runs in a venv. Also version lock it if needed.

Reply:
:tada: thank you @thegreyd! Updated, PTAL :)

Reply:
Looks good ✅